sonar-python-plugin/src/test/java/org/sonar/plugins/python/pylint/PylintSensorTest.java

```diff
@@ -175,6 +177,7 @@ public void issues_other_formats() throws IOException {
     assertThat(first.ruleKey()).hasToString("external_pylint:E1300");
     assertThat(first.type()).isEqualTo(RuleType.CODE_SMELL);
     assertThat(first.severity()).isEqualTo(Severity.MAJOR);
+    assertThat(first.remediationEffort()).isEqualTo(3);
```

Comment:
I think we should also update integration tests to ensure that remediation costs are correctly saved.